GameOfLife/board.h

```diff
@@ -18,7 +16,11 @@ class Board
     void setStateOfCellInTheBoard(int positionInRow, int positionInColumn, bool newStateOfCell) {
         theBoard[positionInRow][positionInColumn] = newStateOfCell; }
     Board();
-    Board(int numberOfRows, int numberOfColumns);
+    Board(int newNumberOfRows, int newNumberOfColumns);
+    int getNumberOfCollumn() const;
+    void setNumberOfCollumn(int value);
+    int getNumberOfRow() const;
+    void setNumberOfRow(int value);
```

Comment:
Yes my OP you have definitely right. There is no reason to create setters for board.